Tests/Integration/IntegrationTestFixtures+UrlLoadBalancing.swift

```diff
@@ -0,0 +1,267 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+import Foundation
+
+@testable import MobileCoin
+import XCTest
+
+extension IntegrationTestFixtures {
+    static func createValidConsensusUrlLoadBalancer()
```

Comment:
Im know Im nit-picking but this isn't really following the Fixture pattern being used elsewhere in the code-base.

Reply:
do some research on other test files, look at 10 or so and you'll get the idea of what fixtures are and how to set them up. They solve for a few things:

- line length, asking for a variable (thats already configured) with `fixture.variable` is alot shorter.
- re-use
- readability

Reply:
Gotcha...  I based all this code on the existing IntegrationTestFixtures.swift file code that was already here, which relied on the 'create' methods as opposed to a more "Fixture" based paradigm...  Will refactor.